Modules/Scripted/Home/Home.py

```diff
@@ -148,6 +154,24 @@ def applyStyle(self, widgets, styleSheetName):
             for widget in widgets:
                 widget.styleSheet = style
 
+    def onVPAWVisualizeButton(self):
+        """
+        Run processing when user clicks "VPAW Visualize" button.
+        """
+        with slicer.util.tryWithErrorDisplay(
```

Comment:
It's fine, I'm okay with keeping it but I'd be a bit happier with it removed if it's all the same to you. If switching modules takes an unusual amount of time then something is wrong. Also, we don't apply this level of error-checking in other steps where it would be much more warranted, so removing it here make everything feel more consistent